### PR TITLE
templates: updating filter position and map switch btn, also list sty…

### LINKED ISSUE #3553: Update component library
check and update comp lib and ensure to include disabled versions of all

### PR DESCRIPTION
**Pull request by phillimorland**
fixes #3553
…ling where appropriate
Additional to story:
- updated style for modules that use non squashed list-items and added additional styling - needs to be checked by PM but as with map change the old styling was looking off so consulted Marcelo and we came up with this.

Following modules have retained the dark module background and sm list items:
- left styling for modules that used smaller list items, e.g. brainstorm both, prioritisation, poll, text commenting, interactive event: these should also probably be adjusted for consistency but should be checked first.

### CODE REVIEW

**Comment by phillimorland** [comment]
I will do a11y in seperate pull request as need to do sme testing first

File: meinberlin/apps/maptopicprio/templates/meinberlin_maptopicprio/maptopic_list.html [hunk 1]
```diff
@@ -12,13 +12,12 @@
...
-        <div class="l-wrapper">
-            <div class="l-center-8">
-                {% include "meinberlin_contrib/includes/map_filter_and_sort.html" with filter=view.filter mode=view.mode %}
-            </div>
+    <div class="l-wrapper u-spacer-bottom u-spacer-top-double">
+        <div class="l-center-8">
+            {% include "meinberlin_contrib/includes/map_filter_and_sort.html" with filter=view.filter mode=view.mode %}
         </div>
     </div>
+    </div>
```
Comment (fuzzylogic2000) [inline meinberlin/apps/maptopicprio/templates/meinberlin_maptopicprio/maptopic_list.html:20]: Isn't this one closing div too much?

File: meinberlin/apps/contrib/templates/meinberlin_contrib/includes/map_filter_and_sort.html [hunk 2]
```diff
@@ -3,16 +3,30 @@
...
     {% else %}
         {% combined_url_parameter request.GET mode='map' as url_par %}
-        <a class="btn btn--light" href="{{ url_par }}" aria-label="{% trans 'View as map' %}">
-            <i class="fa fa-map" aria-hidden="true"></i>
-            {% trans 'View' %}
-        </a>
+        <div class="btn-group__container">
+            <div class="btn-group" role="group">
+                <div class="btn btn--light switch--btn active" aria-label="{% trans 'View as list' %}">
+                  <i class="fa fa-list"></i>
```
Comment (fuzzylogic2000) [inline meinberlin/apps/contrib/templates/meinberlin_contrib/includes/map_filter_and_sort.html:22]: Should the FA-stuff have aria-hidden?

**Review by fuzzylogic2000 — COMMENTED** [review]
Suupernice!
Just some nits in the comments and one in the components library:
Screenshot from 2021-04-26 09-40-18

**Comment by fuzzylogic2000** [comment]
> I will do a11y in seperate pull request as need to do sme testing first

Sorry, so forget about my remark.

**Comment by fuzzylogic2000** [comment]
@phillimorland If you want to also fix the div in your next PR, feel free to merge!

File: meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_list.html [hunk 3]
```diff
@@ -41,13 +39,13 @@
...
     {% else %}
-        <div class="module-content">
+        <div class="module-content--light">
             <div class="l-wrapper">
                 <div class="l-center-8">
                     {% if object_list.count > 0 %}
                         <ul class="u-list-reset">
                             {% for object in object_list %}
-                                {% include "meinberlin_kiezkasse/includes/proposal_list_item.html" with object=object %}
+                                {% include "meinberlin_contrib/includes/proposal_list_item.html" with object=object %}
```
Comment (khamui) [inline meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_list.html:48]: just out of curiosity. What is the difference between the folder `meinberlin_kiezkasse` and `meinberlin_contrib` ?

**Review by khamui — COMMENTED** [review]
Very nice!
Little thing on the proposal mobile view (513px to 800px): wrapping looks a bit off. Discussed offline (mattermost)

File: meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_list.html (hunk 3 above)
Comment (phillimorland) [reply meinberlin/apps/kiezkasse/templates/meinberlin_kiezkasse/proposal_list.html:48]: `meinberlin_kiezkasse`is a module app specific location whereas this template is used in 2 seperate module apps so moved it to contrib so don't have duplicate code

**Review by fuzzylogic2000 — APPROVED** [review]
Looks good to me! @khamui Just merge when you're done reviewing. :)

**Comment by khamui** [comment]
Looking good now 👏

**Pull request merged**